<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemoveDuplicateDialog</class>
 <widget class="QDialog" name="RemoveDuplicateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>723</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>去除重复项</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>指定oto.ini的位置</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="loadOtoWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0">
      <item>
       <widget class="QLineEdit" name="fileNameEdit_open"/>
      </item>
      <item>
       <widget class="QPushButton" name="browseButton_open">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>浏览</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loadButton">
        <property name="text">
         <string>加载</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="countLabel">
       <property name="text">
        <string>还未加载原音设定</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="showOtoListButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>查看列表</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="optionWidget" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="ignoreSpecificSuffixCheckBox">
        <property name="text">
         <string>比较时忽略别名中的特定后缀</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QListWidget" name="suffixListWidget"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="addSuffixButton">
             <property name="text">
              <string>添加</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="deleteSuffixButton">
             <property name="text">
              <string>删除</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="modifySuffixButton">
             <property name="text">
              <string>修改</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ignorePitchSuffixCheckBox">
        <property name="text">
         <string>比较时忽略别名中的音阶名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>音阶范围：</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QComboBox" name="bottomPitchComboBox">
               <item>
                <property name="text">
                 <string>C</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>C#</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>D</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>D#</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>E</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>F</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>F#</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>G</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>G#</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>A</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>A#</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>B</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="bottomPitchSpinBox">
               <property name="minimum">
                <number>-99</number>
               </property>
               <property name="value">
                <number>1</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>到</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QComboBox" name="topPitchComboBox">
               <item>
                <property name="text">
                 <string>C</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>C#</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>D</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>D#</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>E</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>F</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>F#</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>G</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>G#</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>A</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>A#</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>B</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="topPitchSpinBox">
               <property name="minimum">
                <number>-99</number>
               </property>
               <property name="value">
                <number>7</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="caseSensitiveCheckBox">
             <property name="text">
              <string>区分大小写</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>音名字母：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="caseComboBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <item>
              <property name="text">
               <string>大写</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>小写</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>最大重复数量（0为不限制）</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QSpinBox" name="maxSpinBox">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="organizeCheckBox">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;按照顺序重新整理重复项的重复编号，并统一音阶名的音名字母。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>整理重复项</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>整理后音名字母：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="organizeCaseComboBox">
           <item>
            <property name="text">
             <string>大写</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>小写</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="saveOptionWidget" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>保存到</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="saveToSrcRadioButton">
          <property name="text">
           <string>源文件</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">savePathButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="saveToPathRadioButton">
          <property name="text">
           <string>指定路径</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">savePathButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="fileNameEdit_save">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="browseButton_save">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>浏览</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <property name="leftMargin">
         <number>12</number>
        </property>
        <item>
         <widget class="QCheckBox" name="saveDeletedCheckBox">
          <property name="text">
           <string>保存被删除的项到</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="fileNameEdit_save_deleted">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="browseButton_save_deleted">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>浏览</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RemoveDuplicateDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>697</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RemoveDuplicateDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>697</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ignoreSpecificSuffixCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>widget_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ignorePitchSuffixCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>widget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>427</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>485</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveToPathRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>fileNameEdit_save</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>617</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>617</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveToPathRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>browseButton_save</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>617</y>
    </hint>
    <hint type="destinationlabel">
     <x>705</x>
     <y>621</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveDeletedCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>fileNameEdit_save_deleted</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>652</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>656</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveDeletedCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>browseButton_save_deleted</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>652</y>
    </hint>
    <hint type="destinationlabel">
     <x>732</x>
     <y>655</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>caseSensitiveCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_4</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>565</x>
     <y>493</y>
    </hint>
    <hint type="destinationlabel">
     <x>635</x>
     <y>498</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>caseSensitiveCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>caseComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>576</x>
     <y>486</y>
    </hint>
    <hint type="destinationlabel">
     <x>740</x>
     <y>489</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="savePathButtonGroup"/>
 </buttongroups>
</ui>
